draft-packages/table/KaizenDraft/Table/styles.scss

```diff
@@ -35,10 +35,25 @@ $row-height: 60px;
   text-overflow: ellipsis;
   overflow-x: hidden;
   white-space: nowrap;
-  padding: 0 $ca-grid;
-  height: $ca-grid * 1.5;
+  padding: 8px $ca-grid;
```

Comment:
This removal of the fixed height was required to allow for text wrapping.

The 8px is to get the height of the cell roughly the same size as it was before. I was going to do `$ca-grid *0.27`, but I wasn't sure if that defeats the point of the ca-grid constant.

Reply:
How I usually think about this is any [measured alignments](https://liferay.design/articles/best-practices/measured-vs-optical-alignment/) should always use `$ca-grid`, and everything else (optical adjustments, things like this) is okay to use other values as long as it's clear why. In this case, I would drop a comment for the next person.